cmd/lk/room.go

```diff
@@ -784,6 +789,11 @@ func joinRoom(ctx context.Context, cmd *cli.Command) error {
 		return err
 	}
 
+	autoSubscribe, err := extractFlagAsBool(cmd, "auto-subscribe")
+	if err != nil {
+		return err
+	}
```

Comment:
I think we can just read `cmd.Bool("auto-subscribe")` directly, since we are setting a default value for it.